shared/src/main/scala/amf/core/client/platform/model/document/Module.scala

```diff
@@ -6,9 +6,7 @@ import amf.core.client.scala.model.document.{Module => InternalModule}
 
 import scala.scalajs.js.annotation.{JSExportAll, JSExportTopLevel}
 
-/**
-  * Module model class
-  */
+/** Units containing abstract fragments that can be referenced from other fragments */
```

Comment:
`abstract fragments` ??

Reply:
I copied that definition from the client/scala Module class, and agree that is not very clear. How can it be better described?